src/main/java/se/bjurr/gitchangelog/api/GitChangelogApi.java

```diff
@@ -513,6 +514,19 @@ public GitChangelogApi withRedmineUsername(final String string) {
     return this;
   }
 
+  /** Issue Filter Fields for Jira. */
```

Comment:
The comments here does not help me understand what this is.

Reply:
Great point! I updated the comment with a bunch more info/context/examples etc